cumulus_library/databases.py

```diff
@@ -337,7 +370,7 @@ def upload_file(
         return f"s3://{bucket}/{s3_key}"
 
     def close(self) -> None:
-        return self.connection.close()
+        return self.connection.close()  # pragma: no cover
```

Comment:
We never close out the connection in tests?

Reply:
I don't think we have anyplace we're explicitly closing a cursor at the moment, actually.